en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -702,10 +702,14 @@ Eager loading can be used in templates. The following example assumes that `$MyT
 <% end_loop %>
 ```
 
-Eager loading supports all relationships - `has_one`, `belongs_to`, `has_many`, `many_many`, `many_many_through`, `belongs_many_many`.
+Eager loading supports all relationship types.
```

Comment:
No need to specify - all relationship types is all relationship types. Having an explicit list makes it prone to becoming out-of-date.